dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -176,6 +177,16 @@ internal class RumViewScope(
         sendViewUpdate(writer)
     }
 
+    private fun onKeepAlive(
+        event: RumRawEvent.KeepAlive,
+        writer: Writer<RumEvent>
+    ) {
+        delegateEventToChildren(event, writer)
```

Comment:
why do you send it also to the children ? The way I see it this event type is only handled in the RumViewScope.

Reply:
It will possibly be the first event after an action started, and can thus send an action that would otherwise wait a lot longer to be sent. Same goes for resources, it can send resources with missing keys (stop not called).